### PR TITLE
README: add some basic guides

### PR DESCRIPTION
**Pull request by winkyao**
Signed-off-by: winkyao golangwink@gmail.com

as title said.

### CODE REVIEW

**Comment by Hoverbear** [comment]
Hi @winkyao , can you please give some motivation for this repository existing? It seems like all the material here is either duplicates from the TiKV repo, boilerplate, or content that is better off on our website or discourse boards?

**Comment by winkyao** [comment]
@Hoverbear Developers will be more accustomed to using github, and it will be more convenient to discuss directly with issues, which is easier for us to carry out "The TiDB Performance Challenge". In addition, if we operate the SIG, we can use github to display our meeting notes, SIG members, and SIG tasks that can be published via issue directly, just like what k8s does.

**Comment by winkyao** [comment]
Maybe we need to remove the duplicates content from TiKV later. @Hoverbear

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+## Communication
+
+Communication within the TiKV community abides by [TiKV Code of Conduct](./CODE_OF_CONDUCT.md). Here is an excerpt:
+
+> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## License
+
+TiDB Community is under the Apache 2.0 license. See the [LICENSE.md](https://github.com/tikv/tikv/blob/master/LICENSE) file for details.
```
Comment (gingerkidney) [inline README.md]:
```suggestion
TiKV Community is under the Apache 2.0 license. See the [LICENSE.md](https://github.com/tikv/tikv/blob/master/LICENSE) file for details.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at coc@pingcap.com. All
```
Comment (gingerkidney) [inline CODE_OF_CONDUCT.md]: just to make sure if we have the same email with TiDB

**Comment by Hoverbear** [comment]
@winkyao Some content (such as our Code of Conduct and Governance) *must* be in the TiKV repo to fulfill the CII criteria for CNCF Incubating. So they will never be able to be removed.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (Hoverbear) [inline CODE_OF_CONDUCT.md]: We must adopt the CNCF CoC, please copy the file from https://github.com/tikv/tikv/pull/5670

File: become-a-committer.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Become a Committer
```
Comment (Hoverbear) [inline become-a-committer.md]: This should be outlined in our https://github.com/tikv/tikv/blob/master/GOVERNANCE.md, a file which must exist due to the requirements of the CNCF, or https://tikv.org/docs/3.0/contribute/contribute-to-tikv/.

**Comment by Hoverbear** [comment]
@winkyao  We have a discourse (https://forum.tikv.org/) instance which we maintain (and pay for) that is also very suited to these tasks. I recommend we chose one and stick to it, getting rid of the other.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (winkyao) [reply CODE_OF_CONDUCT.md]: Okay, I just linked to this anchor

**Comment by winkyao** [comment]
@Hoverbear PTAL

**Pull request merged**